src/tests/ftest/container/root_container_test.py

```diff
@@ -174,13 +138,15 @@ def verify_multi_pool_containers(self):
                 self.insert_files_and_verify(cont_name)
 
     def verify_create_delete_containers(self, pool, cont_count):
-        """Create multiple containers and multiple multi-mb files
-           in each of them and verify the space usage.
-           Destroy half of the containers and verify the space
-           usage is reclaimed.
+        """Verify multiple pools and containers creation and deletion.
+
+        Create multiple containers and multiple multi-mb files in each of
+        them and verify the space usage.
 
-           Args:
-               cont_count (int): Number of containers to be created.
+        Destroy half of the containers and verify the space usage is reclaimed.
+
+        Args:
+            cont_count (int): Number of containers to be created.
```

Comment:
```suggestion
            int: Number of containers to be created.
```
Is this the right format?

Reply:
For arguments, yes.  Argument names need to be associated with types, as they may be multiple method arguments.  Return values just need a type as the variable assignment is handled by the caller of the method.

Reply:
I see. I got confused with the return value.